lib/smart_proxy_ansible/vcs_cloner_helper.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Proxy
+  module Ansible
+    # Implements VCS-Cloning logic and helper functions
+    class VcsClonerHelper
+      class << self
+        DEFAULT_BASE_PATH = Pathname('/var/lib/foreman-proxy/ansible/roles')
```

Comment:
I'd prefer settings. In `lib/smart_proxy_ansible/plugin.rb` you see a list of `default_settings` and you can provide it there. See the overall review for more on settings.